### PR TITLE
docs(*): update sass documentation

### PR DESCRIPTION
**Pull request by SisIvanova**

### CODE REVIEW

File: sass/color/_functions.scss [hunk 1]
```diff
@@ -337,12 +348,3 @@ $_enhanced-accessibility: false;
...
 }
-
-/// Returns a list of H, S, L values for a given color.
-/// @access public
-/// @param {Color} $color - The color to generate HSL values for.
-/// @group Palettes
-/// @returns { List } - The list of values.
-@function to-hsl($color) {
-    @return (color.hue($color), color.saturation($color), color.lightness($color));
-}
```
Comment (simeonoff) [inline sass/color/_functions.scss:348]: We shouldn't remove this.

**Review by simeonoff — COMMENTED** [review]
See comments

File: sass/color/_functions.scss (hunk 1 above)
Comment (SisIvanova) [reply sass/color/_functions.scss:348]: This function was duplicated so I just removed the second one. See lines 285 - 294

Comment (simeonoff) [reply sass/color/_functions.scss:348]: Sorry, didn't see it.

**Review by simeonoff — APPROVED** [review]

**Pull request merged**